packages/features/bookings/lib/handleNewBooking/test/team-bookings/collective-scheduling.test.ts

```diff
@@ -1087,6 +1090,212 @@ describe("handleNewBooking", () => {
         timeout
       );
 
+      test(
+        `In Team Event type when location is Organizer Default App and User Metadata is {} (Empty Object).`,
```

Comment:
NIT: I think when looking at the title of the test it should be very descriptive what's happening. Could we change this to something like "When event type location is Organizer Default App and user metadata is empty, default to Cal Video".